packages/mock-account-provider/app/routes/shoe-shop.tsx

```diff
@@ -0,0 +1,167 @@
+import { useLocation } from '@remix-run/react'
+import { useEffect, useState } from 'react'
+import { ApiClient } from '~/lib/apiClient'
+import { parseQueryString } from '~/lib/utils'
+
+function AuthorizedView({
+  thirdPartyName,
+  currencyDisplayCode,
+  amount,
+  interactId,
+  nonce
+}: {
+  thirdPartyName: string
+  currencyDisplayCode: string
+  amount: number
+  interactId: string
+  nonce: string
+}) {
+  return (
+    <>
+      <div className='row'>
+        <div className='col-12'>
+          <i className='bi bi-check-circle-fill text-success display-1'></i>
+        </div>
+      </div>
+      <div className='row mt-2'>
+        <div className='col-12'>
+          <p>
+            You gave {thirdPartyName} permission to send {currencyDisplayCode}{' '}
+            {amount.toFixed(2)} out of your account.
```

Comment:
Is the asset scale being applied to the amount here?
On the consent screen, I see:
>You gave undefined permission to send USD 2850.00 out of your account.

For a quote sendAmount of `57.00`

Reply:
Just fixed that

Reply:
I guess we're also missing the `thirdPartyName`
(I'm seeing undefined)